bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PacketProcessorBase.java

```diff
@@ -152,6 +152,8 @@ protected void sendResponseAndWait(int rc, Object response, OpStatsLogger statsL
 
     @Override
     public void safeRun() {
+        requestProcessor.getRequestStats().getWriteThreadQueuedLatency()
+                .registerSuccessfulEvent(MathUtils.elapsedNanos(enqueueNanos), TimeUnit.NANOSECONDS);
```

Comment:
This line will run for both read and write requests. I think that is fine because they share the same queue, but just want to mention it because the name indicates it is only running for writes.